pkg/apis/serving/register.go

```diff
@@ -75,6 +75,14 @@ const (
 	// which Service they are created.
 	ServiceLabelKey = GroupName + "/service"
 
+	// DomainMappingLabelKey is the label key attached to Ingress resources to indicate
+	// which DomainMapping triggered their creation.
+	DomainMappingLabelKey = GroupName + "/domainmapping"
+
+	// DomainMappingNamespaceLabelKey is the label key attached to a Ingress
+	// by a DomainMapping to indicate which namespace the DomainMapping was created in.
+	DomainMappingNamespaceLabelKey = GroupName + "/domainmappingNamespace"
```

Comment:
Is this necessary, given the Ingress is always in the same namespace (per the comment on `MakeIngress`)

Reply:
Yes, you are correct. It is not necessary. Updated.